earthly/rust/config.toml

```diff
@@ -13,14 +13,14 @@ rustflags = [
 linker = "clang"
 rustflags = [
     "-C", "link-arg=-fuse-ld=/usr/bin/mold",
-    # "-C", "target-feature=+crt-static" - Doesn't work for nextest.
+    # "-C", "target-feature=+crt-static" - proc-macro doesn't work with it. `https://github.com/rust-lang/rust/issues/78210`
 ]
 
 [target.aarch64-unknown-linux-gnu]
 linker = "clang"
 rustflags = [
     "-C", "link-arg=-fuse-ld=/usr/bin/mold",
-    "-C", "target-feature=+crt-static"
+    # "-C", "target-feature=+crt-static" - proc-macro doesn't work with it. `https://github.com/rust-lang/rust/issues/78210`
 ]
 
 [target.aarch64-unknown-linux-musl]
```

Comment:
This is the target catalyst-gateway should build with on an apple mac (or any AArch64 based device actually).